src/cljs/replumb/repl.cljs

```diff
@@ -625,18 +683,21 @@
   [opts]
   (let [verbose? (:verbose opts)
         read-file! (:read-file-fn! opts)]
-    (when verbose?
-      (common/debug-prn "Discovering goog/deps.js in" (:src-paths opts)))
-    (doseq [path (:src-paths opts)]
-      (let [goog-deps-path (str (common/normalize-path path) "goog/deps.js")]
-        (read-file! goog-deps-path
-                    (fn [content]
-                      (when content
-                        (do (when verbose?
-                              (common/debug-prn "Found valid" goog-deps-path))
-                            (swap! app-env
-                                   update :goog-provide->path
-                                   merge (goog-deps-map content))))))))))
+    (if read-file!
```

Comment:
This was a bug, there might be no `:read-file-fn!`, only a `load-fn!` so we need to avoid using it